docs/modules/demos/pages/data-lakehouse-iceberg-trino-spark.adoc

```diff
@@ -34,7 +34,7 @@ $ stackablectl demo install data-lakehouse-iceberg-trino-spark
 [#system-requirements]
 == System requirements
 
-The demo was developed and tested on a kubernetes cluster with 10 nodes (4 cores (8 threads), 20GB RAM and 30GB HDD).
+The demo was developed and tested on a kubernetes cluster with 10 nodes (4 cores (8 threads), 20GiB RAM and 30GB HDD).
```

Comment:
During the testing I actually started up two additional nodes since some pods couldn't start because of lacking memory. But I'm not sure if that was due to unfortunate scheduling of pods. Couldn't debug properly because of internet and IONOS connection issues :disappointed:

Reply:
It sounds accurate enough. I think it is only used for warnings, right?

Reply:
It differs so much from the resourceRequests :D ?

Reply:
Only the storage seems to differ. I'm unsure what it _actually_ requires, but reading the above sounds like 30GB total was enough.

```yml
    resourceRequests:
      cpu: "80" # 10 x 4 hyperthreading cores
      memory: 200Gi # 10 x 20 GiB
      pvc: 1Ti # ⚠️ could be dropped to 30 GiB?
```

Reply:
Ignore me, did not take into account the factor 10 due to 10 nodes... :D

Reply:
But there is a discrepancy in the storage.

Reply:
@NickLarsenNZ I'm not sure if the storage of the nodes is used for the pvc or something else externally?

Reply:
PVC would be 300Gb then? Yeah i think just for data storage etc.

Reply:
The `resourceRequests.pvc` value is not currently checked (see: https://github.com/stackabletech/stackable-cockpit/blob/e96139a44309439bc0a60037bf15d621a83371a1/rust/stackable-cockpit/src/platform/cluster/resource_request.rs#L108-L122).

I recommend making the resourceRequests line up with the text, eg:

```yml
resourceRequests:
  pvc: 300Gi
```

Reply:
The 30GB for every k8s nodes (so totaling 300GB) are for storing docker images and emptyDirs and stuff. They are *not* used for PVCs, PVCs are provisioned separately by the csi driver.
So the original content was actually correct ;)